projects/plugins/jetpack/modules/tiled-gallery/math/class-constrained-array-rounding.php

```diff
@@ -69,7 +69,12 @@ private static function get_int_floor_array( $bound_array ) {
 	 * @param int   $adjustment - how much we're adjusting the array.
 	 */
 	private static function adjust_constrained_array( &$bound_array_int, $adjustment ) {
-		usort( $bound_array_int, array( 'self', 'cmp_desc_fraction' ) );
+		usort(
+			$bound_array_int,
+			static function ( $a, $b ) {
+				return strcmp( $b['fraction'], $a['fraction'] );
```

Comment:
I don't think `strcmp` is correct here, as it appears the value is supposed to be numeric.
https://github.com/Automattic/jetpack/blob/fb86ccacc90763a4d0e10c9fd02275a88288f3cf/projects/plugins/jetpack/modules/tiled-gallery/math/class-constrained-array-rounding.php#L57

Reply:
Ah, I was trying to simplify things. That makes sense. I moved back to 6a8cb0fc71d695628cd8c7d6a2778811d807feef